### DISCUSSION TITLE
Can't it play in the background?

### DISCUSSION THREAD

**Discussion by jiafeiyu123** [KRTirtho/spotube#1102] [Q&A] [answered]
Can't it play in the background?

**Comment by KRTirtho** [comment]
Yes

**Comment by nepalivai** [comment, marked as answer]
@jiafeiyu123  Check close behaviour on the setting, it may have been caused because it is on close.
<img width="770" alt="image" src="https://github.com/KRTirtho/spotube/assets/108126089/47c0069c-7564-4a87-8326-5cfc13a7ae73">

**Reply by KRTirtho** [reply]
Yeah, that is a better answer. Sorry @jiafeiyu123 I was a little busy I forgot to upload the image 😅😆

**Reply by nepalivai** [reply]
@KRTirtho no worries!!! always ready to help. Care to highlight it as an answer?

**Reply by nepalivai** [reply]
hey, @KRTirtho stilling waiting for it to be marked as ans.

**Reply by jiafeiyu123** [reply]
I'm very sorry. What I want to ask is that on the IOS system, it seems that it cannot be played in the background.
